.github/workflows/karate.yml

```diff
@@ -31,9 +31,15 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: "3.8"
-      - uses: aws-actions/setup-sam@v1
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
         with:
-          version: 1.37.0
+          java-version: 17
+          distribution: corretto
+      - name: Setup AWS SAM CLI
+        uses: aws-actions/setup-sam@v1
+        with:
+          sam-version: "1.37.0"
```

Comment:
output from github action says parameter `sam-version` is not working and that valid one are just `version`. you should not have to define it anyways. if this parameter is not used it most probably uses the latest version so no need to specify a version at all